covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -0,0 +1,102 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from flask import Flask
+from pathlib import Path
+
+from covid_graphs.country_graph import CountryGraph, GraphType
+from covid_graphs.predictions import prediction_db, PredictionEvent
+
+
+def create_dashboard(
+    data_dir: Path,
+    server: Flask,
+    prediction_event: PredictionEvent,
+    graph_type: GraphType,
+):
+    # TODO(miskosz): Don't use print.
+    print(f"Creating dashboard for {prediction_event.name} {graph_type} graphs.")
+
+    predictions = prediction_db.predictions_for_event(prediction_event)
+
+    # Note: We silently assume there is only one prediction per country.
+    country_graphs = [
+        CountryGraph(data_dir, [country_prediction], graph_type)
+        for country_prediction in predictions
+        if (data_dir / f'{country_prediction.country}.data').is_file()
+    ]
+    country_graphs.sort(key=lambda graph: graph.name)
+
+    app = dash.Dash(
+        name=f'COVID-19 {graph_type}',
+        url_base_pathname=f'/covid19/{prediction_event.name}/{graph_type}/',
+        server=server,
+        external_scripts=[
+            'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-MML-AM_CHTML'
+        ],
+    )
+    app.title = 'COVID-19 predictions'
+    if graph_type != GraphType.Normal:
+        app.title += f' on a {graph_type} graph'
```

Comment:
We should add similar logic to the `html.H1` element - the main headline should reflect the graph type